docs-src/python/how-to-set-timers-in-python.md

```diff
@@ -0,0 +1,15 @@
+---
+id: how-to-set-timers-in-python
+title: How to set Timers in Python
+sidebar_label: Python
+description: To set a Timer in Python, set the `asyncio.sleep()` function.
+tags:
+  - timers
+  - sleep
+---
+
+To set a Timer in Python, set the [`asyncio.sleep()`](https://docs.python.org/3/library/asyncio-task.html#sleeping) function and pass the duration you want to wait in seconds before continuing.
```

Comment:
```suggestion
To set a Timer in Python, set the [`asyncio.sleep()`](https://docs.python.org/3/library/asyncio-task.html#sleeping) function and pass the duration in seconds you want to wait before continuing.
```
Or "...pass the number of seconds you want to wait before continuing" to match the text in other SDKs.